api/public_api_v1.yaml

```diff
@@ -241,12 +241,7 @@ definitions:
 responses: {}
 parameters:
   PaginationRequest:
-    name: cursor
-    in: query
-    type: string
-    minimum: 1
-    description: An opaque identifier that points to the next item in the collection.
-    x-example: 01BX9NSMKVXXS5PSP2FATZM123
+    $ref: 'components.yaml#/x-components/parameters/cursor'
```

Comment:
@gunzip Is the `cursor` still implemented or wip?

Reply:
actually we don't use pagination at all but it's planned